tests/pytest_tests/unit_tests/test_library_public.py

```diff
@@ -259,6 +259,7 @@ def test_library_run():
     "as_dict",
     "update_locked",
     "persist",
+    "merge_locked",
```

Comment:
just making sure that we want to expose this function

Reply:
since we expose `update_locked()` as a public method and we call merge_locked() directly in wandb_run, in order to maintain parity with update_locked() i put this here